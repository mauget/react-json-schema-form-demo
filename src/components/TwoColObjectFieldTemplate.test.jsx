import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import MultiColumnObjectFieldTemplate from './TwoColObjectFieldTemplate';

configure({ adapter: new Adapter() });

describe('MultiColumnObjectFieldTemplate', () => {
    it('should match its snapshot', () => {
        const titleField = jest.fn();
        const properties = [
            { content: { key: 'firstName' } },
            { content: { ket: 'lastName' } },
            { content: { ket: 'age' } },
        ];
        const wrapper = shallow(
            <MultiColumnObjectFieldTemplate
                TitleField={titleField}
                properties={properties}
                title='title'
                description='description'
            />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
