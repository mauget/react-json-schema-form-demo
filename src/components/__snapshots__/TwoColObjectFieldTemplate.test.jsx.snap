// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultiColumnObjectFieldTemplate should match its snapshot 1`] = `ShallowWrapper {}`;
